psi4/src/psi4/cc/ccdensity/ccdensity.cc

```diff
@@ -118,7 +118,7 @@ void td_setup(const struct TD_Params& S);
 void tdensity(const struct TD_Params& S);
 void td_print();
 void oscillator_strength(std::shared_ptr<Wavefunction> wfn, struct TD_Params *S);
-void rotational_strength(MintsHelper &mints, struct TD_Params *S);
+void rotational_strength(std::shared_ptr<Wavefunction> wfn, MintsHelper &mints, struct TD_Params *S);
```

Comment:
If these functions are now accepting a Wavefunction is there a need to pass in a MintsHelper, too?